vertx-db2-client/src/test/java/io/vertx/db2client/tck/DB2TextDataTypeDecodeTest.java

```diff
@@ -0,0 +1,36 @@
+package io.vertx.db2client.tck;
+
+import java.math.BigDecimal;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.vertx.db2client.junit.DB2Resource;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.sqlclient.tck.TextDataTypeDecodeTestBase;
+
+@RunWith(VertxUnitRunner.class)
+public class DB2TextDataTypeDecodeTest extends TextDataTypeDecodeTestBase {
+    @ClassRule
+    public static DB2Resource rule = DB2Resource.SHARED_INSTANCE;
+
+    @Override
+    protected void initConnector() {
+        connector = ClientConfig.CONNECT.connect(vertx, rule.options());
+    }
+
+    @Test
+    public void testDecimal(TestContext ctx) {
+        // In DB2 NUMERIC type == DECIMAL type so always return BigDecimal
+        testDecodeGeneric(ctx, "test_decimal", BigDecimal.class, BigDecimal.valueOf(12345));
```

Comment:
can we use the `Numeric` Vert.x type to replace `BigDecimal` like other projects do? it's useful to be used in a polyglot environment.

Reply:
good point, I've pushed another commit that resolves this so now `testDecimal` and `testNumeric` are not overridden in the DB2 TCK tests